packages/gafl-webapp-service/src/pages/licence-details/licence-start-date/__tests__/licence-start-date.spec.js

```diff
@@ -1,11 +1,4 @@
-import {
-  LICENCE_START_DATE,
-  LICENCE_LENGTH,
-  LICENCE_START_TIME,
-  DATE_OF_BIRTH,
```

Comment:
I don't think we should be making arbitrary format changes to each others code.

Reply:
I think this was picked up by prettier-standard.  It's slightly odd as the whole point is that it should be consistent for us both so this sort of reformatting change should never occur.